### DISCUSSION TITLE
TwoFactorAuth v3

### DISCUSSION THREAD

**Discussion by NicolasCARPi** [RobThree/TwoFactorAuth#131] [General]
This thread is for general discussion about this release.

**WARNING** read the [CHANGELOG](https://github.com/RobThree/TwoFactorAuth/blob/master/CHANGELOG.md#version-3x) as there are some breaking changes!

@willpower232 @RobThree can you tag 3.0.0 and do a release please?

**Comment by willpower232** [comment]
Looks like its missing a few links to PRs and perhaps a mention of the fix for throwing a curl exception?

**Reply by NicolasCARPi** [reply]
What do you think about tagging a release now?

**Reply by RobThree** [reply]
I'm away for the weekend, but if both of you agree then you have my blessing 👍

**Reply by willpower232** [reply]
Carved out a few minutes to wrap up the changelog https://github.com/RobThree/TwoFactorAuth/pull/135 but then yeah, lets do it.

**Comment by willpower232** [comment]
Completed it https://github.com/RobThree/TwoFactorAuth/releases/tag/v3.0.0
